pep-513.rst

```diff
@@ -9,7 +9,7 @@ Status: Draft
 Type: Informational
 Content-Type: text/x-rst
 Created: 19-Jan-2016
-Post-History: 19-Jan-2016
+Post-History: 24g-Jan-2016
```

Comment:
> On which note I guess it might be a good idea to include a link somewhere in the references to the manylinux-discuss archives where we debated stuff initially.

Is that typical? Should I also link to the distutils-sig discussion?